st2actions/st2actions/container/base.py

```diff
@@ -129,10 +129,17 @@ def _do_run(self, runner, runnertype_db, action_db, liveaction_db):
             LOG.debug('Action "%s" completed.' % (action_db.name), extra=extra)
 
             # Always clean-up the auth_token
-            updated_liveaction_db = self._update_live_action_db(liveaction_db.id, status,
-                                                                result, context)
-            executions.update_execution(updated_liveaction_db)
+            try:
+                LOG.debug('Setting status: %s for liveaction: %s', status, liveaction_db.id)
+                updated_liveaction_db = self._update_live_action_db(liveaction_db.id, status,
+                                                                    result, context)
+            except:
+                error = 'Cannot update LiveAction object for id: %s, status: %s, result: %s.' % (
+                    liveaction_db.id, status, result)
+                LOG.exception(error)
+                raise
```

Comment:
Slightly confused - this doesn't really change the behavior since we `raise` here. If updating the live action fails, we will re-throw the exception which means execution won't be updated. The only difference right now is that we log the message.

Is this intentional?